web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -0,0 +1,114 @@
+import useResource from '../../../custom/api/useResource';
+import { AccountObject } from '../../../models/AccountModel';
+import Intl from '../../../i18n/i18n';
+import { useLocation } from 'react-router-dom';
+import { parseForm } from '../../../utils/parseForm';
+import { useState, ChangeEvent, FormEvent } from 'react';
+import {
+  formatStandard,
+  reverseFormatStandard,
+} from '../../../utils/MassaFormating';
+import { SendForm } from './SendForm';
+import { SendConfirmation } from './SendConfirmation';
+
+interface IErrorObject {
+  amount?: string;
+  fees?: string;
+  recipient?: string;
+  error?: string;
+}
+
+function Send() {
+  const { state } = useLocation();
+  const nickname = state.nickname;
+  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
+  const account = accounts.find((account) => account.nickname === nickname);
+  const balance = parseInt(account?.candidateBalance ?? '0') / 10 ** 9;
```

Comment:
What's that logic!? 🤔

Reply:
we decomposed it like that 
```ts
  const unformattedBalance = account?.candidateBalance  ?? '0'; 
  const balance = parseInt(unformattedBalance) / 10 ** 9;
```
unformattedBalance is the account.candidateBalance if candidateBalance is found, else 0
balance is the parsed int with the right number of decimals

Reply:
This:
```
js
...parseInt(unformattedBalance) / 10 ** 9;
```
wont return!? `1e-9`. 🤔 

Is that we expect!?